cryptoprotocol/src/test/java/org/cryptimeleon/incentivesystem/cryptoprotocol/Helper.java

```diff
@@ -0,0 +1,53 @@
+package org.cryptimeleon.incentivesystem.cryptoprotocol;
+
+import org.cryptimeleon.craco.sig.sps.eq.SPSEQSignature;
+import org.cryptimeleon.incentivesystem.cryptoprotocol.model.IncentivePublicParameters;
+import org.cryptimeleon.incentivesystem.cryptoprotocol.model.Token;
+import org.cryptimeleon.incentivesystem.cryptoprotocol.model.keys.provider.ProviderKeyPair;
+import org.cryptimeleon.incentivesystem.cryptoprotocol.model.keys.user.UserKeyPair;
+
+import java.math.BigInteger;
+
+public class Helper {
```

Comment:
Might deserve a docstring because of generic class name.

Reply:
I will add that

Reply:
Outdated comment